backend/server.py

```
@@ -6724,14 +6864,65 @@ async def workflow_orchestrator_approve(
+
+
+@app.get("/api/workflow/orchestrator/status")
+async def workflow_orchestrator_status(
+    user: dict = Depends(get_current_user),
+):
+    """Return orchestrator queue depth, active runs, and supervisor state (#522)."""
+    orchestrator = get_workflow_orchestrator()
+    owner_id = None if _wfo_is_admin(user) else _wfo_resolve_user_id(user)
+    runs = orchestrator.list_runs(limit=200, owner_id=owner_id)
+
+    queue_status = {max_concurrent: 2, active: 0, queued: 0}
```

Ruff (0.15.15)

[error] Undefined name `max_concurrent`

(F821)